sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -889,54 +891,67 @@ def upload_manifest(
             digest = response_headers['Docker-Content-Digest']
             if not _validate_digest(data, digest):
                 raise ValueError("The digest in the response does not match the digest of the uploaded manifest.")
-        except ValueError:
+        except Exception:
             if repository is None or manifest is None:
                 raise ValueError("The parameter repository and manifest cannot be None.")
             raise
         return digest
 
     @distributed_trace
-    def upload_blob(self, repository: str, data: IO, **kwargs) -> str:
+    def upload_blob(self, repository: str, data: IO[bytes], **kwargs) -> Tuple[str, int]:
```

Comment:
Should we include the change in return type as a breaking change in the changelog?

Reply:
To make sure we are on same page, upload_blob was not in last GA release. It is a breaking change from last beta.

Reply:
I thought we documented breaking changes from beta -> beta version as well.

Reply:
Yes. I pointed it out just to make sure we are on the same page that it is a beta -> beta breaking.

(Breaking GA will cause trouble and likely be pushed back. :))